2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_setup.py:_flush():69] setting env: {'project': 'NMT', 'entity': 'jungwoohan72', 'root_dir': '/home/jungwoo/Coursework/AI504/Project2', 'sweep_id': 'xo69487m', 'run_id': 'xc1f7lkh', 'sweep_param_path': '/home/jungwoo/Coursework/AI504/Project2/wandb/sweep-xo69487m/config-xc1f7lkh.yaml'}
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_init.py:_log_setup():342] Logging user logs to /home/jungwoo/Coursework/AI504/Project2/wandb/run-20211205_172009-xc1f7lkh/logs/debug.log
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_init.py:_log_setup():343] Logging internal logs to /home/jungwoo/Coursework/AI504/Project2/wandb/run-20211205_172009-xc1f7lkh/logs/debug-internal.log
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_init.py:init():375] calling init triggers
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_init.py:init():380] wandb.init called with sweep_config: {'attention_heads': 4, 'batch_size': 64, 'decoder_layers': 3, 'dropout': 0.2, 'emb_dim': 32, 'encoder_layers': 3, 'ffn_dim': 32, 'lr': 0.0001, 'nepochs': 50, 'patience': 10}
config: {}
2021-12-05 17:20:09,534 INFO    Thread-2  :20334 [wandb_init.py:init():424] starting backend
2021-12-05 17:20:09,535 INFO    Thread-2  :20334 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-05 17:20:09,541 INFO    Thread-2  :20334 [backend.py:ensure_launched():135] starting backend process...
2021-12-05 17:20:09,547 INFO    Thread-2  :20334 [backend.py:ensure_launched():139] started backend process with pid: 20358
2021-12-05 17:20:09,552 INFO    Thread-2  :20334 [wandb_init.py:init():429] backend started and connected
2021-12-05 17:20:09,554 INFO    Thread-2  :20334 [wandb_run.py:_config_callback():826] config_cb None None {'attention_heads': 4, 'batch_size': 64, 'decoder_layers': 3, 'dropout': 0.2, 'emb_dim': 32, 'encoder_layers': 3, 'ffn_dim': 32, 'lr': 0.0001, 'nepochs': 50, 'patience': 10}
2021-12-05 17:20:09,554 INFO    Thread-2  :20334 [wandb_init.py:init():477] updated telemetry
2021-12-05 17:20:09,554 INFO    Thread-2  :20334 [wandb_init.py:init():500] communicating current version
2021-12-05 17:20:10,146 INFO    Thread-2  :20334 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-05 17:20:10,146 INFO    Thread-2  :20334 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-12-05 17:20:10,379 INFO    Thread-2  :20334 [wandb_init.py:init():540] starting run threads in backend
2021-12-05 17:20:14,193 INFO    Thread-2  :20334 [wandb_run.py:_console_start():1584] atexit reg
2021-12-05 17:20:14,194 INFO    Thread-2  :20334 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-12-05 17:20:14,194 INFO    Thread-2  :20334 [wandb_run.py:_redirect():1463] Redirecting console.
2021-12-05 17:20:14,195 ERROR   Thread-2  :20334 [wandb_run.py:_redirect():1522] Failed to redirect.
Traceback (most recent call last):
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1515, in _redirect
    out_redir.install()
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/jungwoo/anaconda3/envs/AI504/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2021-12-05 17:20:14,196 INFO    Thread-2  :20334 [wandb_init.py:init():565] run started, returning control to user process
2021-12-05 17:20:36,393 INFO    Thread-2  :20334 [wandb_run.py:finish():1254] finishing run jungwoohan72/NMT/xc1f7lkh
